src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs

```diff
@@ -161,6 +161,22 @@ public override Task<IActionResult> RemoveConnectionFromGroup([RegularExpression
             return Task.FromResult(NotFound() as IActionResult);
         }
 
+        public override Task<IActionResult> RemoveConnectionFromAllGroups([RegularExpression(HubPattern)] string hub, string connection)
+        {
+            if (!ModelState.IsValid)
+            {
+                return Task.FromResult(BadRequest() as IActionResult);
```

Comment:
why not `return Task.FromResult<IActionResult>(BadRequest());`

Reply:
fixed

Reply:
Simply follow the code style of other methods in the same file(`SignalRServiceEmulatorWebApi.cs`).  For example: [L84](https://github.com/Azure/azure-signalr/blob/417fa6407b70b78c33b18ab3a0f21cbf0dcc84f9/src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs#L84) , [L263](https://github.com/Azure/azure-signalr/blob/417fa6407b70b78c33b18ab3a0f21cbf0dcc84f9/src/Microsoft.Azure.SignalR.Emulator/Controllers/SignalRServiceEmulatorWebApi.cs#L263)

Reply:
Let's have another PR to batch update them